applications/machine_learning/README.rst

```diff
@@ -174,15 +182,23 @@ The application supports the following development kits:
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
-   :rows: thingy52_nrf52832, nrf52840dk_nrf52840
+   :rows: thingy52_nrf52832, thingy53_nrf5340_cpuapp, nrf52840dk_nrf52840
 
 The available configurations use only built-in sensors or the simulated sensor signal.
 There is no need to connect any additional components to the board.
 
-.. note::
-   The Thingy:52 does not have the J-Link debug IC and the application configuration does not use a bootloader.
-   Use an external debugger to program the firmware.
-   See :ref:`zephyr:thingy52_nrf52832` documentation for details.
+Thingy:52
```

Comment:
```suggestion
Bootloader for Thingy:52

Reply:
Can we call it `Programming Thingy:52` and  `Programming Thingy:53`. The Thingy:52 actually does not have a bootloader so the section title would be a bit misleading.